pkg/kev/converter/kubernetes/utils.go

```diff
@@ -955,16 +951,6 @@ func contains(strs []string, s string) bool {
 	return i < len(strs) && strs[i] == s
 }
 
-// runtimeObjectConvertToTarget converts runtime object into a target
-func runtimeObjectConvertToTarget(o runtime.Object, target interface{}) error {
```

Comment:
Just checking.. Wasn't this func in use at all?

Reply:
Nope. No where.

Unless `utils.go` has an external entity calling it - nothing was using `runtimeObjectConvertToTarget`.

Reply:
After more research nothing uses it.